### ISSUE TITLE
Remove duplicated imports

### ISSUE THREAD

**Issue by andreea-pasare** [OP-TED/model2owl#213]
This replicates in all *.rdf files. Please see the output from the latest rc: [ePO/implementation/ePO_core/owl_ontology/ePO_core.rdf at v4.2.0-rc.1 · OP-TED/ePO](https://github.com/OP-TED/ePO/blob/v4.2.0-rc.1/implementation/ePO_core/owl_ontology/ePO_core.rdf) 

![image](https://github.com/user-attachments/assets/5ecfa01c-ca88-4868-a127-24ed3e4f7458)

In the image above we have the m8g namespace that comes from various core vocabularies prefixes: cccev, clv, cpov, etc.

**Comment by rousso** [comment]
@cristianvasquez Please label this as a bug if it is one.

**Comment by gkostkowski** [comment]
The issue was resolved (PR #216). The change will be included in the next model2owl release.

**Solution details**:
RDF/XML artefacts generated by model2owl are now post-processed using `rdfpipe` serializer. That serializer has been already used to generate TTL files and now is also used to restructure RDF/XML files. That ensures that the returned RDF/XML files don't contain the reported redundant `owl:imports` statements. 

Using the serializer also results in a more compact RDF/XML files as the serializer represents data in the optimal way. The following excerpts shows the difference.

Former RDF/XML representation:
```xml
<owl:DatatypeProperty rdf:about="http://data.europa.eu/a4g/ontology#hasAbnormallyLowTenders"/>
<rdf:Description rdf:about="http://data.europa.eu/a4g/ontology#hasAbnormallyLowTenders">
  <skos:prefLabel xml:lang="en">Has abnormally low tenders</skos:prefLabel>
</rdf:Description>
<rdf:Description rdf:about="http://data.europa.eu/a4g/ontology#hasAbnormallyLowTenders">
  <skos:definition xml:lang="en">Tenders received that were found irregular and non-acceptable due to an abnormally low price or cost. Additional Information The definition implies that abnormally Low Tender Lots are Inadmissible Tender Lots. WG Approval 28/07/2020 (epo:SubmissionStatisticalInformation)</skos:definition>
</rdf:Description>
<rdf:Description rdf:about="http://data.europa.eu/a4g/ontology#hasAbnormallyLowTenders">
  <rdfs:isDefinedBy rdf:resource="http://data.europa.eu/a4g/ontology#core"/>
</rdf:Description>
```

New RDF/XML representation:
```xml
<rdf:Description rdf:about="http://data.europa.eu/a4g/ontology#hasAbnormallyLowTenders">
  <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#DatatypeProperty"/>
  <skos:prefLabel xml:lang="en">Has abnormally low tenders</skos:prefLabel>
  <skos:definition xml:lang="en">Tenders received that were found irregular and non-acceptable due to an abnormally low price or cost. Additional Information The definition implies that abnormally Low Tender Lots are Inadmissible Tender Lots. WG Approval 28/07/2020 (epo:SubmissionStatisticalInformation)</skos:definition>
  <rdfs:isDefinedBy rdf:resource="http://data.europa.eu/a4g/ontology#core"/>
</rdf:Description>
```
